### ISSUE TITLE
Week 3 + Week 4

### ISSUE THREAD

**Issue by farzaa** [TeamMoodGitHub/Felipe-Term-1#3]
Whats up mate!

So at this point you should have a pretty good handle on apis, web requests, and how js works! This week I'd like you to work on a more real web app.

I want to see a webpage where I input my summoner name and it throws out some of my stats from the **last week**. Things like kills, deaths, games won, rank fluctuation, etc.

I also want the web page to check if I'm in a game **every 15 seconds**. If I am in a game, it'll say "in game". But, lets say I have the web page open already and then I get in a game, the message should change from "not in game" to "in game".

The above two things should be ezpz since its all just API stuff. What I want you to do is start getting into Firebase since fro your final app you'll need a database. So, on your webpage when a summoner name is input, I also want you to send some of those stats like kills/deaths to your database. This may be a struggle in the beginning but I'll be here!

Basically, I'm having you do this as an exercise so we when you go to actually make the Desktop app, it won't feel so hard since you already did it here. Plus, electron is all HTML/CSS/JS so that helps to :).

**Comment by felipe-linares** [comment]
TODO:

- Use fetch with client-side stored summonerName
- Use loop get information for each individual game
- Learn to use game information to firebase
